.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto-merge
+
+on: pull_request_target
+
+
+jobs:
+  auto-merge:
+    if: github.actor == 'dependabot'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
FTR I'm almost sure this is unnecessary
```suggestion
```

Reply:
Hmm, it's in their examples, and their own workflow file:
https://github.com/ahmadnassri/action-dependabot-auto-merge/blob/master/.github/workflows/auto-merge.yml

Reply:
I guess that depends on whether they use GH API or `git push`...

Reply:
Looking at the code, I think it actually posts a comment. That will 100% not work because API calls produced by the embedded token don't cause any events...

Reply:
There are 1 or 2 other attempts, if anything else looks more promising:
https://github.com/aio-libs/aiohttp/issues/5665#issuecomment-864552750

Reply:
Well, I could fix this with an app token I think.
Integrate this by example: https://github.com/aio-libs/aiohttp/blob/a341986/.github/workflows/backport.yml#L23-L33.

Reply:
I don't even need to add a repo secret — it's already set up on the org level. So for as long as you use the same secret var names in the workflow it'll work.